Plugin doesn’t work with JDK 11

And in contrast to #554 just adding the module won’t fly.

    [ERROR] Failed to execute goal com.github.kongchen:swagger-maven-plugin:3.1.7:generate (default) on project footool-services: Execution default of goal com.github.kongchen:swagger-maven-plugin:3.1.7:generate failed: A required class was missing while executing com.github.kongchen:swagger-maven-plugin:3.1.7:generate: javax/xml/bind/annotation/XmlRootElement

I have also experienced problems with the plugin when running with Java11 - though, not related to the JAXB.
In my situation, the problem is with SQLException:
`[ERROR] Failed to execute goal com.github.kongchen:swagger-maven-plugin:3.1.7:generate (default) on project com.infor.ion.oneview.engine.rest: Execution default of goal com.github.kongchen:swagger-maven-plugin:3.1.7:generate failed: A required class was missing while executing com.github.kongchen:swagger-maven-plugin:3.1.7:generate: java/sql/SQLException`

Currently, the workaround is only to disable the plugin (setting the execution phase to _none_). 

Do you have any ideas how to overcome this, please?

@breun , yes - that's the weird part - the java.sql package didn't changed in the j9-j11 changes at all... 
The project does not use modules. It's a maven-based application. See the attachment, please, for the plugin config.
[plugin_maven_config.txt](https://github.com/kongchen/swagger-maven-plugin/files/2718376/plugin_maven_config.txt)

I also noticed quite cryptic (for me at least) error message in the end of the output log:
```
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
```
Could this be any hint, please?

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,7 @@ language: java
 
 jdk:
   - oraclejdk8
+  - openjdk11
 
 before_install:
 

--- a/pom.xml
+++ b/pom.xml
@@ -112,11 +112,6 @@
             <version>2.2.11</version>
         </dependency>
         <dependency>
-            <groupId>javax.activation</groupId>
-            <artifactId>activation</artifactId>
-            <version>1.1.1</version>
-        </dependency>
-        <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-artifact</artifactId>
             <version>${maven.version}</version>

--- a/pom.xml
+++ b/pom.xml
@@ -97,6 +97,26 @@
 
     <dependencies>
         <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-core</artifactId>
+            <version>2.2.11</version>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.xml.bind</groupId>
+            <artifactId>jaxb-impl</artifactId>
+            <version>2.2.11</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.activation</groupId>
+            <artifactId>activation</artifactId>
+            <version>1.1.1</version>
+        </dependency>
+        <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-artifact</artifactId>
             <version>${maven.version}</version>

--- a/pom.xml
+++ b/pom.xml
@@ -80,6 +80,7 @@
         <json-unit.version>1.17.0</json-unit.version>
         <version.mockito-core>2.18.3</version.mockito-core>
         <version.hamcrest-library>1.3</version.hamcrest-library>
+        <version.jaxb>2.3.1</version.jaxb>
 
         <!-- Plugin versions -->
         <version.maven-release-plugin>2.5.3</version.maven-release-plugin>
@@ -99,17 +100,14 @@
         <dependency>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
-            <version>2.2.11</version>
-        </dependency>
-        <dependency>
-            <groupId>com.sun.xml.bind</groupId>
-            <artifactId>jaxb-core</artifactId>
-            <version>2.2.11</version>
+            <version>${version.jaxb}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>com.sun.xml.bind</groupId>
             <artifactId>jaxb-impl</artifactId>
-            <version>2.2.11</version>
+            <version>${version.jaxb}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.maven</groupId>

